### PR TITLE
Edmundson Ch. 12

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/13-chapter-12.adoc [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-About the year 1690, he, with many more, saw the fulfilling of his prophecy,
-mentioned in the last section.
+[.chapter-synopsis]
+* His frequently exposing his own life to save his neighbours,
+* and standing his ground in those perilous times, till his house was set on fire,
+* himself and two sons led away to be murdered by the Irish raparees;
+* yet by Divine Providence were delivered out of their hands
+* About the year 1690, he, with many more, saw the fulfilling of his prophecy,
+* mentioned in the last section
+* //Not sure what to do with this one!!
```
Comment (Henderjay) [inline journal/modernized/13-chapter-12.adoc]: Ok, so this is the one that we were slacking about. I would maybe arrange this one like this:
```ASCIIDOC
[.chapter-synopsis]
* His frequently exposing his own life to save his neighbours, and standing his ground in those perilous times
* His house set on fire; himself and two sons led away to be murdered by the Irish Raparees
* Delivered out of their hands by Divine Providence
* About the year 1690, he, with many more, see the fulfilling of his prophecy, mentioned in the last section
```
I don't know if Jared told you, but you don't have to worry about capitalizing the key words of these synopsis sections any more. The linter will do it automatically.👍🏻

File: journal/modernized/13-chapter-12.adoc [hunk 2]
```diff
@@ -356,12 +358,13 @@ that if they had killed me and my sons,
 they would have killed all the Irish they met with.
 From Mullingar I came to Jane Barcroft`'s, near Edenderry,
 and from there to Mountmelick to my wife, where many were glad to see me again.
-We shifted for house-room as veil as we could, the town being thronged with soldiers,
+//I noticed "Mountmelick" above was hyphenated in one of the ch. synopses, don't know if that's important for continuity, and I don't remember which chapter!! :)
```
Comment (Henderjay) [inline journal/modernized/13-chapter-12.adoc]: I would just leave it for now. I think Jared will standardize this programmatically in the near future.

File: journal/modernized/13-chapter-12.adoc [hunk 3]
```diff
@@ -378,14 +381,15 @@ because this would hinder them from harbouring their kindred and countrymen,
 who were raparees, as they had frequently done before,
 therefore they got eight or nine bloody raparees,
 to lie in ambush between Mount-melick and Rossenallis in order to kill me,
+//Hyphenated here too??
```
Comment (Henderjay) [inline journal/modernized/13-chapter-12.adoc]: yeah, just leave it here too.  We'll figure it out and change them all at once with the linter.

File: journal/original/13-chapter-12.adoc [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
-yet by Divine Providence were delivered out of their hands.--About the year 1690, he,
-with many more, saw the fulfilling of his prophecy, mentioned in the last section.
+[.chapter-synopsis]
+* His frequently exposing his own life to save his neighbours,
+* and standing his ground in those perilous times, till his house was set on fire,
+* himself and two sons led away to be murdered by the Irish raparees;
+* yet by Divine Providence were delivered out of their hands
+* About the year 1690, he, with many more, saw the fulfilling of his prophecy,
+* mentioned in the last section
+* //Not sure what to do with this one!!
```
Comment (Henderjay) [inline journal/original/13-chapter-12.adoc]: I would do this the same as above.

**Review by Henderjay — CHANGES_REQUESTED** [review]
Ok, I showed how I would do this chapter synopsis... just fyi.  Also mentioned that we can fix all of the Mountmellicks with the linter without you having to worry about finding or fixing them.

File: journal/modernized/13-chapter-12.adoc (hunk 1 above)
Comment (maijapapaya) [reply journal/modernized/13-chapter-12.adoc]: Yes, he let me know 😄

**Comment by Henderjay** [comment]
all good, merging.

**Pull request merged**